api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -558,14 +556,14 @@ function democracyProposalReducer(
 function democracyReferendumReducer(
   network,
   proposal,
+  parameter,
```

Comment:
??? First you need to calculate it right?

I am not even sure the refenrendums have this field. I will check

Reply:
yeah i wasn't able to verify last night because my local API was having issues - just pushed and went to bed